org-asterion/config/Pulumi.dev.yaml

```diff
@@ -0,0 +1,22 @@
+config:
+  aws:defaultTags:
+    tags:
+      environment: dev
+  aws:region: ap-southeast-2
+  org-asterion:accountId: 822258672282
+  org-asterion:currentOrgName: shawng
```

Comment:
Are these values valid for asterion? The look like personal account.

Reply:
`currentOrgName` actually relates to the pulumi org, not the aws asterion account. This was required for any stack references, which are now no longer part of the solution so I've removed all references of this.